inference-engine/src/legacy_api/src/convert_function_to_cnn_network.cpp

```diff
@@ -49,8 +49,11 @@
 #include "transformations/utils/utils.hpp"
 #include "transformations/rt_info/fused_names_attribute.hpp"
 #include "transformations/rt_info/primitives_priority_attribute.hpp"
+#include <cpp/ie_cnn_network.h>
```

Comment:
I think this is local include. Consider `<...>` -> `"..."`.

Reply:
Corrected.